### DISCUSSION TITLE
Can bladebit support  Plotter XCC?

### DISCUSSION THREAD

**Discussion by guitengyue** [Chia-Network/bladebit#148] [Q&A] [answered]
Can bladebit support  Plotter XCC?

**Comment by harold-b** [comment]
Please open ask questions in discussions instead of issues (Converting this to a discussion).

**Reply by harold-b** [reply]
What is Plotter XCC?

**Reply by papaloveyou** [reply]
XCC is Chives .
Hope to add features.

**Reply by papaloveyou** [reply]
XCC use k29-k31

**Reply by guitengyue** [reply]
k29-k31，port chives = 9699

**Comment by harold-b** [comment, marked as answer]
Thanks for clarifying.
No, there won't be any support below k32.
